src/devices/nm-acd-manager.c

```diff
@@ -117,21 +118,15 @@ static const char *
 _acd_error_to_string (int error)
```

Comment:
the name does not seem optimal. Positive numbers are not really errors, are they?

Reply:
Yes, N_ACD_E_PREEMPTED and N_ACD_E_INVALID_ARGUMENT are errors.

_N_ACD_E_SUCCESS of course is not, but I think _acd_error_to_string() is fine as a name for a function that translates error codes to string.